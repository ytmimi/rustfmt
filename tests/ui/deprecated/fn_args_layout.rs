// rustfmt-fn_args_layout: Tall
