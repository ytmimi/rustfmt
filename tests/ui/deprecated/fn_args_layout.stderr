Warning: the `fn_args_layout` option is deprecated. Use `fn_params_layout`. instead
